src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.ts

```diff
@@ -239,7 +296,7 @@ export class InstructorCoursesPageComponent implements OnInit {
     const modalRef: NgbModalRef = this.modalService.open(CourseSoftDeletionConfirmModalComponent);
     modalRef.result.then(() => {
       this.courseService.binCourse(courseId).subscribe((course: Course) => {
-        this.loadInstructorCourses();
+        this.moveCourseToRecycleBin(courseId);
```

Comment:
`course` now will have the true `deletionTimestamp` in the backend. Manually assigning a fake `deletionTimestamp` might cause data inconsistency in the future.